domains/pabitrabanerjee.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "PB2204",
+           "email": "rockstarpabitra2204@gmail.com",
+           "discord": "1091585182974345366"
+        },
+    
+        "record": {
+            "CNAME": "12.14.243.127"
```

Comment:
I believe this is meant to be an A record; IPs are are meant for A records, not CNAMEs.